dist-workspace.toml

```diff
@@ -4,10 +4,12 @@
 [workspace]
 members = ["cargo:."]
 
-# Config for 'cargo dist'
+# Config for 'dist'
 [dist]
-# The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
-cargo-dist-version = "0.23.0"
+# The preferred dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.28.0"
```

Comment:
I realize this conflict is coming for me soon :see_no_evil: #90

Reply:
Unfortunately it's not so simple, each version needs the release.yml regenerated.